src/plugins/discord/testUtil.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+
+export const testChannel = (id: number) => ({
+  id: id,
+  name: "testChannelName",
+  type: 0,
+});
+
+export const testChannelObj = (id: number) => ({
+  id: id,
+  name: "testChannelName",
+  type: "GUILD_TEXT",
+});
+
+export const testMessage = (id: number) => ({
+  id: id,
+  author: {id: 2},
+  timestamp: "2020-03-03T23:35:10.615000+00:00",
+  content: "Just going to drop this here",
+  reactions: [{emoji: {id: 1, name: "testemoji"}}],
+  mentions: [{id: 4, username: "testuser"}],
+});
+
+export const testMember = (userId: number) => ({
+  user: {
+    id: userId,
+    username: "username",
+    discriminator: "disc",
+    bot: true,
+  },
+  nick: "nickname",
+  roles: ["test role"],
+});
```

Comment:
It's important to use Flow types for example data like this. It will help you validate the example data is actually compliant with the models we're expecting. And will help prevent mistakes if we update these models.

In this case I would suggest using the function return type for that. E.g.
```diff
- const testChannel = (id: number) => ({ ... });
+ const testChannel = (id: number): Model.Channel => ({ ... });
```

In this case, it should actually fail for every function here, because a `Snowflake` is a string, and we're using `number` instead.

Reply:
Great point! Updated.

Reply:
I think this might have been undone in some places by moving the functions? :smile:
I'll comment inline on the ones I can spot that aren't typed yet.